example.yaml

```diff
@@ -162,6 +162,28 @@ trace_max_length_bytes: 16384
 # you think Veneur needs more room to keep up with all packets.
 read_buffer_size_bytes: 2097152
 
+# How many metrics to include in the body of each POST to Datadog. Veneur
+# will post multiple times in parallel if the limit is exceeded.
+flush_max_per_body: 25000
```

Comment:
how did this get added in this PR, but the FlushMaxPerBody above in config.go didn't? IDGI.